src/android/SplashScreen.java

```diff
@@ -255,6 +259,10 @@ public void run() {
                 }
                 splashDialog.setContentView(splashImageView);
                 splashDialog.setCancelable(false);
+                if (isForceFullScreen()) {
```

Comment:
I was thinking about the same thing myself. The problem I think is how the full screen is implemented. The way it's done there is the old-style (pre-4.1) way and I don't think it hides the navigation bar.

I unfortunately don't have access to a device that old so I didn't want to cause an error with older phones.

With some experimentation we could probably combine them.